packages/models-library/src/models_library/services_constants.py

```diff
@@ -11,3 +10,40 @@
 }
 
 FILENAME_RE = r".+"
+
+
+# e.g. simcore/services/comp/opencor
+SERVICE_KEY_RE: Final[re.Pattern[str]] = re.compile(
+    r"^simcore/services/"
+    r"(?P<type>(comp|dynamic|frontend))/"
+    r"(?P<subdir>[a-z0-9][a-z0-9_.-]*/)*"
+    r"(?P<name>[a-z0-9-_]+[a-z0-9])$"
+)
+# e.g. simcore%2Fservices%2Fcomp%2Fopencor
+SERVICE_ENCODED_KEY_RE: Final[re.Pattern[str]] = re.compile(
+    r"^simcore%2Fservices%2F"
+    r"(?P<type>(comp|dynamic|frontend))%2F"
+    r"(?P<subdir>[a-z0-9][a-z0-9_.-]*%2F)*"
+    r"(?P<name>[a-z0-9-_]+[a-z0-9])$"
+)
+
+DYNAMIC_SERVICE_KEY_RE: Final[re.Pattern[str]] = re.compile(
+    r"^simcore/services/dynamic/"
+    r"(?P<subdir>[a-z0-9][a-z0-9_.-]*/)*"
+    r"(?P<name>[a-z0-9-_]+[a-z0-9])$"
+)
+DYNAMIC_SERVICE_KEY_FORMAT = "simcore/services/dynamic/{service_name}"
+
+COMPUTATIONAL_SERVICE_KEY_RE: Final[re.Pattern[str]] = re.compile(
+    r"^simcore/services/comp/"
+    r"(?P<subdir>[a-z0-9][a-z0-9_.-]*/)*"
+    r"(?P<name>[a-z0-9-_]+[a-z0-9])$"
+)
+COMPUTATIONAL_SERVICE_KEY_FORMAT: Final[str] = "simcore/services/comp/{service_name}"
+
+
+LATEST_INTEGRATION_VERSION: Final[str] = "1.0.0"
+
+
+#
+ANY_FILETYPE: Final[str] = "data:*/*"
```

Comment:
these are actually the only constants in this module (with line 35)... it is a bit ill named no? or maybe the regexes should move to basic_regex